ffn/utils.py

```diff
@@ -45,7 +46,7 @@ def memoize(f, refresh_keyword="mrefresh"):
     return decorator.decorator(_memoize, f)
 
 
-def parse_arg(arg):
+def parse_arg(arg: Any):
```

Comment:
really should be `Union[str, List[str], Tuple[str]]`

Reply:
Right.  Also replacing all `|` with `Union` due to python version

Reply:
we can bump the minimum python version to `3.9` if needed, `3.8` is end of life in October

Reply:
`|` was added in python 3.10 if I'm not mistaken